docs/source/contents/kubernetes/scaling/index.md

```diff
@@ -0,0 +1,55 @@
+# Scaling
+
+## Models
+
+Models can be scaled by setting their replica count, e.g.
+
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Model
+metadata:
+  name: iris
+spec:
+  storageUri: "gs://seldon-models/scv2/samples/mlserver_1.2.3/iris-sklearn"
+  requirements:
+  - sklearn
+  memory: 100Ki
+  replicas: 3
+```
+
+The number of replicas will need not to exceed the replicas of the Server the model is scheduled to.
+
+## Servers
+
+Servers can be scaled by setting their replica count, e.g.
+
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Server
+metadata:
+  name: mlserver
+  namespace: seldon
+spec:
+  replicas: 4
+  serverConfig: mlserver
+```
+
+Models scheduled to a server can only scale up to the server replica count.
+
+## Internal Components
+
+Seldon core v2 runs with several control and dataplane components. The scaling of these resurces is discussed below:
+
+- Pipeline gateway.
+    - This pipeline gateway handles REST and gRPC synchronous requests to Pipelines. It is stateless and can be scaled based on traffic demand.
+- Model gateway.
+    - This component pulls model requests from Kafka and sends them to inference servers. It can be scaled up to the partition factor of your Kafka topics. At present we set a uniform partition factor for all topics in one installation of Seldon Core V2.
+- Dataflow engine.
+    - The dataflow engine runs KStream topologies to manage Pipelines. It can run as multiple replicas and the scheduler will balance Pipelines to run across it with a consistent hashing load balancer with each Pipeline managed up to the partition factor of Kafka (presently hardwired to one).
```

Comment:
```suggestion
    - The dataflow engine runs KStream topologies to manage Pipelines. It can run as multiple replicas and the scheduler will balance Pipelines to run across it with a consistent hashing load balancer. Each Pipeline is managed up to the partition factor of Kafka (presently hardwired to one).
```